### PR TITLE
Remove idp metadata with sp-entity-id from home

### PR DESCRIPTION
**Pull request by MKodde**
The EB homepage advertised the possibility to open IdP metadata with the certificate of a specified SP. This feature was not longer supported in Engine. At least, I was not able to reproduce the behavior described for this feature.

There was not a lot to clean up, so only the homepage links have been removed.

See: https://www.pivotaltracker.com/story/show/168249086

### CODE REVIEW

**Comment by MKodde** [comment]
Nope, there is a clear distinction between IdP and IdPs metadata. The first is no longer used in context with the sp-entity-id, the latter is used for wayf construction.

File: src/OpenConext/EngineBlockFunctionalTestingBundle/Features/FrontPage.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  In order to inform users of EngineBlock's capabilities
+  As EngineBlock
+  I want to display a front page with links to my metadata endpoints and other features
+
+  Background:
+    Given an EngineBlock instance on "vm.openconext.org"
+      And no registered SPs
+      And no registered Idps
+
+  Scenario: The user can see all available metadata links
```
Comment (pablothedude) [inline src/OpenConext/EngineBlockFunctionalTestingBundle/Features/FrontPage.feature]: Consider splitting the tests the way the frontpage is grouped.

Consider asserting the number of links to detect if entries are added. That way someone will need to fix this and is notified of the existence of the tests.

**Comment by pablothedude** [comment]
A good call to add the test!

File: src/OpenConext/EngineBlockFunctionalTestingBundle/Features/FrontPage.feature (hunk 1 above)
Comment (MKodde) [reply src/OpenConext/EngineBlockFunctionalTestingBundle/Features/FrontPage.feature]: Good points!
1. Updated the display order of the step definitions to match that of the page
2. Added an expected number of links assertion

**Pull request merged**